cmd/opampsupervisor/generated_package_test.go

```diff
@@ -1,12 +1,10 @@
-// Copyright The OpenTelemetry Authors
-// SPDX-License-Identifier: Apache-2.0
+// Code generated by mdatagen. DO NOT EDIT.
 
 package main
 
 import (
-	"testing"
-
 	"go.uber.org/goleak"
+	"testing"
```

Comment:
you need to run `make generate` which will run gci after generating, fixing the imports. Or fix the imports by hand, reverting change of line 7 to 9.

Reply:
Thanks! Fixed. Although now the unit tests fail, but it seems like an unrelated error for the Datadog exporter that hopefully goes away after a retry.

```
Error Trace:	/home/runner/work/opentelemetry-collector-contrib/opentelemetry-collector-contrib/exporter/datadogexporter/integrationtest/integration_test.go:501
            				/opt/hostedtoolcache/go/1.23.8/x64/src/runtime/asm_amd64.s:1700
Error:      	Received unexpected error:
            	cannot start pipelines: failed to start "otlp" receiver: listen tcp 127.0.0.1:42791: bind: address already in use
Test:       	TestIntegrationLogs
```